app/javascript/flavours/polyam/features/notifications_v2/components/notification_reaction.tsx

```diff
@@ -52,7 +52,7 @@ export const NotificationReaction: React.FC<{
       accountIds={notification.sampleAccountIds}
       statusId={notification.statusId}
       timestamp={notification.latest_page_notification_at}
-      count={notification.notifications_count}
+      count={notification.sampleAccountIds.length}
```

Comment:
One issue with this is that the text will only ever show "[..]and 7 others" with more than 8 reactions as the account IDs are capped at 8.

Reply:
Changed the count query instead.